sources/updater/updog/src/main.rs

```diff
@@ -897,56 +846,66 @@ mod tests {
         let first_bound = Utc::now();
         u.waves.insert(100, first_bound);
         assert!(
-            u.update_wave(1).unwrap() == Wave::Initial { end: first_bound },
+            u.update_wave(1).unwrap()
+                == Wave::Initial {
+                    end_time: first_bound,
+                    end_seed: 100
+                },
             "Expected to be 0th wave"
         );
-        assert!(u.jitter(1).is_none(), "Expected immediate update");
         assert!(
-            u.update_wave(101).unwrap() == Wave::Last { start: first_bound },
+            u.update_wave(101).unwrap()
+                == Wave::Last {
+                    start_time: first_bound,
+                    start_seed: 100
```

Comment:
Should the 100 match the 101?  (Not sure I fully understand what this is doing)

(Same below with 200/100, the 150, the 201/200...)

Reply:
101 places it in the last wave. The test is making sure the update belongs to the last wave if seed is 101 and the initial wave goes up to seed 100. Same with the others.

Reply:
After some offline discussion, we decided that `early_wave` test is redundant because tests in `update_metadata` already cover testing wave ownership of seeds.